.buildkite/pipeline-resource-definitions/kibana-serverless-security-solution-quality-gate-explore.yml

```
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: kibana_serverless_security_solution_quality_gate_explore
```

Since then, I've adjusted the generation script that this should follow the suggested pattern by eng-prod/dev-tooling - although, I wonder if these names will hit a limit or not.

```suggestion
  name: buildkite-pipeline-kibana-serverless-security-solution-quality-gate-explore
```